app/src/main/java/ru/sonya/week3/CatAdapter.kt

```diff
@@ -7,14 +7,35 @@ import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 
-class CatAdapter (private val catList:ArrayList<FunCats>): RecyclerView.Adapter<CatAdapter.CatViewHolder>()
+class CatAdapter (): RecyclerView.Adapter<CatAdapter.CatViewHolder>()
 {
+   var  catList:MutableList<FunCats> = mutableListOf()
+       set(value) {
+           field = value
+       }
```

Comment:
```suggestion
       var catList = listOf<FunCats>()
        set(value) {
            field = value
            notifyDataSetChanged()
        }
```
Последняя строчка используется для оповещения recyclerView об изменении списка элементов